Readme.md

```diff
@@ -52,22 +52,28 @@ Flags:
   -h, --help       Show context-sensitive help.
       --debug      Enable debug mode.
       --dry-run    Show what would be run.
+      --systems="config/systems.yaml"
+                   path to the systems config file
+      --repos="config/repositories.yaml"
+                   path to the repositories config file
+      --daemons="config/daemons.yaml"
+                   path to the daemons config file
 
 Commands:
-  init
-    Initialize cluster.
-
-  deploy
+  deploy [<only> ...]
     Deploy to current context.
 
-  undeploy
+  undeploy [<only> ...]
     Undeploy from current context.
 
-  make <command>
+  make <command> [<only> ...]
     Run make [COMMAND] in every repository.
 
-  install
-    Install daemons.
+  install [<node> ...]
+    Install daemons (EXPERIMENTAL).
```

Comment:
can probably remove the experimental part

Reply:
That's fresh output from nnf-deploy.  We can adjust that in a separate PR.